scripts/tls-common.sh

```diff
@@ -17,6 +17,8 @@ export IPXE_KEY_FILE=/certs/ipxe/tls.key
 export RESTART_CONTAINER_CERTIFICATE_UPDATED=${RESTART_CONTAINER_CERTIFICATE_UPDATED:-"false"}
 
 export MARIADB_CACERT_FILE=/certs/ca/mariadb/tls.crt
+export BMC_CACERT_PATH=/certs/ca/bmc
+export BMC_CACERT_FILE=/conf/bmc-tls.pem
```

Comment:
I don't follow the naming or content of these variables. We define path to one place, but the file is elsewhere under `/conf` ?

Reply:
OK. Can we fix the naming then a bit, something like `BMC_CACERTS_PATH` and `BMC_CACERT_FILE` ? Just make a minor difference to say they're on purpose not used like a common pattern PATH + FILE = FULLPATH.

Reply:
Fixed